src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -38,194 +43,190 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach with TableDrivenPro
+      s"$defaultJsonName.json",
       docxFileIdInList ++ jsonFileIdInList,
       "docx checksum & json checksum don't match"
     ),
     (
       nonMatchingChecksumValue,
       nonMatchingChecksumValue,
-      s"$defaultFileName.docx",
-      nonMatchingFileName,
+      defaultDocxTitle,
+      nonMatchingJsonName,
       docxFileIdInList ++ jsonFileIdInList,
       "docx checksum, json checksum & json file name don't match"
     ),
     (
       nonMatchingChecksumValue,
       nonMatchingChecksumValue,
-      nonMatchingFileName,
-      s"$defaultFileName.json",
+      nonMatchingDocxTitle,
+      s"$defaultJsonName.json",
       docxFileIdInList ++ jsonFileIdInList,
-      "docx checksum, json checksum & docx file name don't match"
+      "docx checksum, json checksum & docx file title don't match"
     ),
     (
       nonMatchingChecksumValue,
       nonMatchingChecksumValue,
-      nonMatchingFileName,
-      nonMatchingFileName,
+      nonMatchingDocxTitle,
+      nonMatchingJsonName,
       docxFileIdInList ++ jsonFileIdInList,
-      "docx checksum, json checksum, docx file name & json file name don't match"
+      "docx checksum, json checksum, docx file title & json file name don't match"
     )
   )
 
   "handleRequest" should "return an error if the asset is not found in Dynamo" in {
-    val testUtils = new ExternalServicesTestUtils(dynamoServer)
-    testUtils.stubGetRequest(testUtils.emptyDynamoGetResponse)
-    val testLambda = testUtils.TestLambda()
+    stubGetRequest(emptyDynamoGetResponse)
+    val testLambda = TestLambda()
     val ex = intercept[Exception] {
-      testLambda.handleRequest(testUtils.standardInput, testUtils.outputStream, null)
+      testLambda.handleRequest(standardInput, outputStream, null)
     }
-    ex.getMessage should equal(s"No asset found for ${testUtils.assetId} from ${testUtils.batchId}")
+    ex.getMessage should equal(s"No asset found for ${assetId} from ${batchId}")
 
     testLambda.verifyInvocationsAndArgumentsPassed(0, 0, 0, 0)
   }
 
   "handleRequest" should "return an error if the Dynamo entry does not have a type of 'Asset'" in {
-    val testUtils = new ExternalServicesTestUtils(dynamoServer)
-    testUtils.stubGetRequest(testUtils.dynamoGetResponse.replace(""""S": "Asset"""", """"S": "ArchiveFolder""""))
-    testUtils.stubPostRequest(testUtils.emptyDynamoPostResponse)
-    val testLambda = testUtils.TestLambda()
+    stubGetRequest(dynamoGetResponse.replace(""""S": "Asset"""", """"S": "ArchiveFolder""""))
+    stubPostRequest(emptyDynamoPostResponse)
+    val testLambda = TestLambda()
     val ex = intercept[Exception] {
-      testLambda.handleRequest(testUtils.standardInput, testUtils.outputStream, null)
+      testLambda.handleRequest(standardInput, outputStream, null)
     }
-    ex.getMessage should equal(s"Object ${testUtils.assetId} is of type ArchiveFolder and not 'Asset'")
+    ex.getMessage should equal(s"Object ${assetId} is of type ArchiveFolder and not 'Asset'")
```

Comment:
Same here. There's a few of them in this file, mostly `${assetId}` and `${batchId}`